crates/evm/src/executor/inspector/cheatcodes/fork.rs

```diff
@@ -4,17 +4,19 @@ use crate::{
     executor::{
         backend::DatabaseExt, fork::CreateFork, inspector::cheatcodes::ext::value_to_token,
     },
-    utils::{b160_to_h160, RuntimeOrHandle},
+    utils::{h160_to_b160, ru256_to_u256, u256_to_ru256, RuntimeOrHandle},
 };
 use ethers::{
     abi::{self, AbiEncode, Token, Tokenizable, Tokenize},
-    prelude::U256,
     providers::Middleware,
-    types::{Bytes, Filter, H256},
+    types::{Bytes, Filter, U256 as eU256},
 };
 use foundry_abi::hevm::{EthGetLogsCall, RpcCall};
 use foundry_common::ProviderBuilder;
-use revm::EVMData;
+use revm::{
+    primitives::{B256, U256},
```

Comment:
use alloy types

Reply:
Done on https://github.com/foundry-rs/foundry/pull/5768/commits/efd575160101d98dfe34cc372b32bebda657146d and https://github.com/foundry-rs/foundry/pull/5768/commits/69e06eed172cb708ca042e8266b6f9c64a96ac01